src/locales/en/translation.json

```diff
@@ -1351,54 +1397,12 @@
     "promotion1_3": "Ready to participate? Head over to the event page by clicking the button below!",
     "promotion1_cta": "Go to event"
   },
-  "fastBtcDialog": {
-    "title": "Top Up FastBTC",
-    "subtitle": "Deposit BTC to receive rBTC in your wallet",
-    "note": "Always be sure to check the BTC deposit address as FastBTC gets updated",
-    "limits": {
-      "title": "Deposit limits:",
-      "min": "MIN: {{amount}}",
-      "max": "MAX: {{amount}}",
-      "btc": "BTC"
-    },
-    "instructions": {
-      "title": "Instructions:",
-      "point1": "Do not deposit anything other than BTC",
-      "point2": "Do not send more BTC than the MAX limit",
-      "point3": "Allow up to 60 mins for the transaction to process",
-      "point4": "If rBTC is not visible on your wallet after 60 mins open a ",
-      "point4LinkText": "support ticket",
-      "chainId": "Make sure your wallet is set to correct network. Liquality Wallet requires dapp refresh to detect network change."
-    },
-    "fiatDialog": {
-      "title": "Buy <0>RBTC</0> with fiat",
-      "explanation1": "Transak allows you to buy BTC which is then converted to <0>RBTC</0> and sent to your wallet",
-      "explanation2": "Make sure you do not exceed Sovryn deposit limits as Transak allows purchases above our max deposit limits",
-      "instructions": {
-        "point1": "Do not buy more BTC than the MAX limit above",
-        "point2": "Allow  15 mins for the transaction to process",
-        "chainId": "Make sure your wallet is set to correct network. Liquality Wallet requires dapp refresh to detect network change."
-      }
-    },
-    "qr": {
-      "title": "Send BTC to this address:",
-      "generating": "Generating..."
-    },
-    "buttonBTC": "Generate deposit address",
-    "buttonFiat": "Get rBTC with fiat",
-    "transaction": {
-      "title": "Transaction Details",
-      "status": "Status {{status}}",
-      "from": "From:",
-      "to": "To:",
-      "fee": "Tx Fee:",
-      "hash": "Tx Hash:",
-      "close": "Close",
-      "address": "Address",
-      "pending": "pending",
-      "confirmed": "confirmed",
-      "sent": "sent"
-    }
+  "transakDialog": {
+    "title": "Buy <0>RBTC</0> with fiat",
+    "explanation1": "Transak allows you to buy BTC which is then converted to <0>RBTC</0> and sent to your wallet",
```

Comment:
I think Transak service now allows to purcharse RBTC directly. So no need to convert from BTC to rBTC

Reply:
Yes, will request copy guys for new texts